examples/plotBounds.R

```diff
@@ -1,9 +1 @@
 # examples
-plotBounds(boundsPredprob(
```

Comment:
@audreyyeoCH can you please first merge in `main` into this feature branch? Because currently these examples are deleted, but they should be there (because https://github.com/Genentech/phase1b/blob/main/examples/plotBounds.R has them)

Reply:
This file still has "deleted" contents, but should not, because these examples were just added in the previous PR to `main`?